### PR TITLE
Fix a bug which swapped 'File' and 'Edit' menus

### PR DESCRIPTION
**Pull request by imagesurgery**
Screenshot
On nuke 11.3v4 the File and Edit menus were being drawn in reverse order. This commit adds the same commands but retains the correct menu order.

I presume it is because this command is _finding_ the menu, whereas the original code was _declaring_ it directly.

### CODE REVIEW

**Comment by imagesurgery** [comment]
Aaand... I'm closing this because I noticed what I did wrong. I source a lot of my code in init.py, but this explicitly needs to be sourced after the UI begins to be processed in menu.py. Because of this, your code was creating a menu called `edit` before `file` existed, and adding the commands in. When I 'fixed' it, I actually didn't, but my module loader bypassed it because it was throwing an error, which being in a try block, didn't stop nuke or other tools from loading.

So.

In short, it's all my fault, and I have a solution. I also shall now check my terminal for errors after changes to ensure they're not actually errors.
